src/Profile/ProfileScreen.js

```diff
@@ -170,25 +172,25 @@ export default function ProfileScreen({ navigation }) {
           Alert.alert(err.error, err.message);
         }
       });
+      const updatedUser = {
+        ...currentUser, email, address, phoneNumber: phoneNum,
+      };
+      serviceUpdateUser(updatedUser);
     } else {
       Alert.alert('Please fill out all fields.');
     }
-  }
+  };
 
-  function onCancel() {
-    console.log('cancel');
+  const onCancel = () => {
     setEditMode(false);
     setTitle('Edit');
-  }
-
-  const [shouldShow, setShouldShow] = useState(true);
-
-  const [title, setTitle] = useState('Edit');
+  };
 
   return (
+    // TODO: fix how the keyboard view blocks the text fields
```

Comment:
styling needs to be improved, the keyboard doesn't make it easy to see the changes made to each field and the buttons edit, save, cancel, and drawer menu are not sized/spaced correctly, it is essential you test how a user would see this screen